### DISCUSSION TITLE
Why is Quasar so inconsistent and annoying?

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [quasarframework/quasar#16445] [General - Components / Directives / etc] [answered]
Hi,

I've noticed there are lots of inconsistencies in Quasar. The roadmap has been stale for a year now. No new components. Buggy `QSelect`. Lol.

Why is Quasar such a mess?

Thanks!

**Comment by pdanpdan** [comment, marked as answer]
> Why is Quasar such a mess?

I don't know. But if you feel it is then don't use it, I'm sure the project will not be very affected.
It uses Vue anyway and as Vue is dying (https://github.com/vuejs/core/discussions/9138) it makes no sense to use something that uses it.

**Comment by metalsadman** [comment]
thanks for the good laugh lol.
